draft/2023-12-20-this-week-in-rust.md

```diff
@@ -162,43 +148,46 @@ Rusty Events between 2023-12-20 - 2024-01-17 🦀
     * [**Crafting Interpreters in Rust Collaboratively**](https://www.meetup.com/charlottesville-rust-meetup/events/297687485/)
 * 2024-01-03 | Virtual (Indianapolis, IN, US) | [Indy Rust](https://www.meetup.com/indyrs/)
     * [**Indy.rs - with Social Distancing**](https://www.meetup.com/indyrs/events/wqzhftygccbfb)
-
-### Asia
-
-* 2023-12-16 | Delhi, IN | [Rust Delhi](https://www.meetup.com/rustdelhi/)
-    * [**Meetup #4**](https://www.meetup.com/rustdelhi/events/297652586/)
+* 2024-01-09 | Virtual (Dallas, TX, US) | [Dallas Rust](https://www.meetup.com/dallasrust/)
+    * [**Last Tuesday**](https://www.meetup.com/dallasrust/events/fvdtgtygccbmb/)
+* 2024-01-11 | Virtual (Charlottesville, NC, US) | [Charlottesville Rust Meetup](https://www.meetup.com/charlottesville-rust-meetup/)
```

Comment:
@andrewpollack We received a note to fix the city for the Charlottesville Rust Meetup. Should be "Charlottesville, VA, US" (VA = Virginia instead of NC = North Carolina).

How is the meetups list generated? Unsure where that comes from.